docusaurus.config.js

```diff
@@ -134,7 +136,20 @@ const config = {
        ] 
       }
     ]
-  ]
+  ],
+  themes: [
+    [
+      require.resolve("@getcanary/docusaurus-theme-search-pagefind"),
+      { 
+        styles: { 
+          "--canary-color-primary-c": 0.1, 
+          "--canary-color-primary-h": 270, 
+        }, 
+        includeRoutes: ["**/*"], 
+        excludeRoutes: ["next", ...versions].map(v => `**/${v}/**`),
```

Comment:
Displaying 2-3 versions is well supported([video](https://github.com/microsoft/fast/pull/7031)), but for N versions, it is not well supported.

I think this is plausible workaround for now, and will come back later.